profile_library/shelly/Shelly EM/model.json

```diff
@@ -0,0 +1,28 @@
+{
+  "measure_description": "The measured device fw: 1.14.1-rc1, Eco Mode enabled, Wifi Client enabled, LED disabled, one measurement clamp connected. The result represents the average value of measurements collected from the measurement device API for the following scenarios: dumb device/baseline consumption (~4.9W), the the relay turned off, and the relay turned on.",
+  "measure_method": "manual",
+  "measure_device": "Shelly PM Mini gen3",
+  "measure_device_firmware": "20241011-114503/1.4.4-g6d2a586",
+  "measure_settings": {
+    "SAMPLE_COUNT": 200,
+    "SLEEP_TIME": 0.5
+  },
+  "name": "Shelly EM",
+  "aliases": [
+    "SHEM"
+  ],
+  "discovery_by": "entity",
+  "device_type": "smart_switch",
+  "only_self_usage": false,
+  "calculation_strategy": "fixed",
+  "standby_power": 0.598,
+  "fixed_config": {
```

Comment:
Please use the new recommended format as explained for scenarios here:
https://docs.powercalc.nl/library/device-types/smart-switch/#smart-switch-with-built-in-powermeter

So you can use `standby_power_on` for this and get rid of `fixed_config`

I will migrate all the other smart switch profiles to this definite profile format soon.